app/QuestionOptions.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+//es kpoxes QuestionOption
+// modelneri anunner@ hognaki ches grum
+// bazayum table-neri anunnern es hognaki grum
+class QuestionOption extends Model
+{
+    public $table="question_options";
```

Comment:
btw, if you decide to keep this...
```suggestion
    public $table = 'question_options';
```